products/batch_exports/backend/tests/temporal/destinations/test_s3_batch_export_workflow.py

```diff
@@ -223,6 +223,64 @@ async def read_json_file_from_s3(s3_compatible_client, bucket_name, key) -> list
     return data[0]
 
 
+MetricKind = t.Literal["success", "cancellation", "failure"]
+MetricName = t.Literal["succeeded", "canceled", "failed"]
```

Comment:
Maybe a bit redundant here, but just in case we ever have 1 metric kind with multiple names.